warehouse/query-core/src/main/java/datawave/query/attributes/UniqueGranularity.java

```diff
@@ -52,6 +82,16 @@ public static UniqueGranularity of(String name) {
                 return UniqueGranularity.TRUNCATE_TEMPORAL_TO_HOUR;
             case "MINUTE":
                 return UniqueGranularity.TRUNCATE_TEMPORAL_TO_MINUTE;
+            case "SECOND":
+                return UniqueGranularity.TRUNCATE_TEMPORAL_TO_SECOND;
+            case "MILLISECOND":
+                return UniqueGranularity.TRUNCATE_TEMPORAL_TO_MILLISECOND;
+            case "TENTH_OF_HOUR":
+                return UniqueGranularity.TRUNCATE_TEMPORAL_TO_TENTH_OF_HOUR;
+            case "MONTH":
+                return UniqueGranularity.TRUNCATE_TEMPORAL_TO_MONTH;
+            case "YEAR":
+                return UniqueGranularity.TRUNCATE_TEMPORAL_TO_YEAR;
```

Comment:
It may be worth considering sorting these cases either alphabetically, or by time unit largest to smallest. From `ALL` to `MILLISECOND` it's in logical descending order, and then tenth of hour, month, and year are thrown in somewhat piecemeal. 

I don't really have a preference, it seems off. But I'm also fine accepting this as-is.

Reply:
I'll sort these out in the next commit, descending for some continuity.